app/code/Magento/Catalog/Ui/Component/Listing/Columns/ProductActions.php

```diff
@@ -58,7 +58,7 @@ public function prepareDataSource(array $dataSource)
                         'catalog/product/edit',
                         ['id' => $item['entity_id'], 'store' => $storeId]
                     ),
-                    'ariaLabel' => __('Edit ') . $item['name'],
+                    'ariaLabel' => __('Edit ') . (isset($item['name']) ? $item['name'] : ''),
```

Comment:
Parenthesis are meaningless here.

You could simplify using : 

```suggestion
                    'ariaLabel' => __('Edit ') . $item['name'] ?? '',
```

A better approach could probably be to define it directly in the translation, but the translation needs to be add

```
                    'ariaLabel' => __('Edit %1', $item['name'] ?? ''),
```

Reply:
Thanks, will have a look later this week